tests/phpunit/elementor/app/import-export/processes/test-import.php

```diff
@@ -475,6 +475,82 @@ public function test_get_manifest__kit_library() {
 		}
 	}
 
+	public function test_init_import_session__get_kit_thumbnail_from_manifest() {
+		$import_settings = [
+			'referrer' => 'kit-library',
+		];
+		$module = new Module();
+
+		$module->import_kit( __DIR__ . '/../mock/kit-library-manifest-with-thumbnail.zip', $import_settings );
+
+		$import_sessions = get_option( Module::OPTION_KEY_ELEMENTOR_IMPORT_SESSIONS );
+		$this->assertEquals( 'manifest-thumbnail', $import_sessions[ $module->import->get_session_id() ]['kit_thumbnail'] );
+	}
+
+	public function init_import_session__get_empty_kit_thumbnail() {
+		$import_settings = [
+			'referrer' => 'kit-library',
+		];
+		$module = new Module();
+
+		$module->import_kit( __DIR__ . '/../mock/kit-library-manifest-only.zip', $import_settings );
+
+		$import_sessions = get_option( Module::OPTION_KEY_ELEMENTOR_IMPORT_SESSIONS );
+		$this->assertEmpty( $import_sessions[ $module->import->get_session_id() ]['kit_thumbnail'] );
```

Comment:
maybe assert an empty string, to make sure there is always a string
(`assertEmpty` will pass for nullish values)